src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala

```diff
@@ -88,12 +87,7 @@ private[bigquery] class DirectBigQueryRelation(
             .setRequestedStreams(getNumPartitions)
             .setReadOptions(readOptions)
             .setTableReference(tableReference)
-            // TODO(aryann): Once we rebuild the generated client code, we should change
-            // setUnknownFields() to use setShardingStrategy(ShardingStrategy.BALANCED). The
-            // BALANCED strategy is public at the moment, so setting it using the unknown fields
-            // API is safe.
```

Comment:
Some comment about what balanced means (link to any public docs?) would still be good